src/protobuf/select.rs

```diff
@@ -1,4 +1,4 @@
-// This file is generated by rust-protobuf 2.5.0. Do not edit
+// This file is generated by rust-protobuf 2.6.1. Do not edit
```

Comment:
It seems that `cargo` upgrades the `rust-protobuf` dependency automatically.
It should be OK and here is another example https://github.com/pingcap/tipb/commit/535e1abaa330653c8955cb3484261ea9f229e926#diff-9ab65b665f1b3a0f209eddc266d5a70eL1 .
@breeswish Please help to confirm it.

Reply:
Why is there 5000+ lines of difference 🤔

Reply:
It seems that the rust code format changed...
Format of this PR is different from [the previous PR](https://github.com/pingcap/tipb/commit/535e1abaa330653c8955cb3484261ea9f229e926#diff-9ab65b665f1b3a0f209eddc266d5a70eL1).
So which one is better...

Reply:
@BusyJay PTAL, it is ok with this change?

Reply:
kvproto uses 2.0.6, I suggest to keep using the same version.

Reply:
OK, I will generate rust code again;

Reply:
@BusyJay  Should we commit locks?

Reply:
I generated rust code again with the previous PR's protobuf version(2.5.0).
PTAL @BusyJay

Reply:
@breeswish No, we don't. Better lock it inside `Cargo.toml`.
@qw4990 Have you tested it with TiKV? I'm not sure whether it breaks anything if the version doesn't match.

Reply:
I tried to do some tests, but there are some things (about `protoc` version and `prost`) wrong in TiKV to let it can't upgrade `tipb`(https://github.com/tikv/tikv/issues/4864), and TiKV has hard-written a specific version of `tipb` now... @BusyJay

Reply:
Since TiKV locks `tipb` to a specific version, I prefer to merge this PR first to let TiDB(https://github.com/pingcap/tidb/pull/10753) can use this new function and then solve problems in TiKV later.